### PR TITLE
Dark background color for the header

### PR DESCRIPTION
**Pull request by dmitryduev**
The current light blue header seems to grab all the attention. Compare:
ab

A darker header also seems more appropriate for dark mode:
dark-mode

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -37,12 +37,14 @@
     "raw-loader": "4.0.1",
     "react": "16.13.1",
     "react-datepicker": "3.1.3",
+    "react-diff-viewer": "^3.1.1",
```
Comment (acrellin) [inline package.json]:
```suggestion
    "react-diff-viewer": "3.1.1",
```

**Review by acrellin — APPROVED** [review]

**Review by dannygoldstein — APPROVED** [review]

**Review by guynir42 — APPROVED** [review]

**Comment by guynir42** [comment]
image

**Review by kmshin1397 — APPROVED** [review]

**Comment by profjsb** [comment]
I'm in favor of this, however it would be good to darken background in dark mode of the table, survey name cells, buttons, etc. The scrollbar also looks bad in dark mode with the (too light) outline.

You sure we dont also want night vision mode?
image
Joke.

**Review by profjsb — CHANGES_REQUESTED** [review]
See comments inline.

**Comment by dmitryduev** [comment]
@profjsb regarding dark mode, @kmshin1397 filed an issue for that https://github.com/skyportal/skyportal/issues/1305, so I think it's well beyond the scope of this PR to find and adjust all the places that need it -- it's just about the header bg color. Sorry, can't see any inline comments..?

**Review by profjsb — APPROVED** [review]

**Pull request merged**